monitoring/PROMETHEUS.md

```diff
@@ -78,6 +78,24 @@ We are currently using the following sets of metrics with Kafka-Exporter:
 * Message consume per minute;
 * Partitions per Topic.
 
+## Kong API Gateway Exporter
+
+As ``Kong`` officially allows the integration, the process becomes simpler. It is only necessary to declare ``job`` in the Prometheus configuration file to get metrics regarding cloud services to manage, monitor and scale application programming interfaces and microservices.
+
+Available metrics:
```

Comment:
I like this `Available metrics` term that you use here.
As a suggestion, I think you could put this same term on the 73, 51, 34 and 15 lines to standardize the file.